<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_image">
      <property name="title">
       <string>图像信息解译</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="1" column="3">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="image_4">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>4.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="image_8">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>8.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="image_6">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>6.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="image_5">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>5.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="image_3">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>3.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_last">
               <property name="text">
                <string>上一批图像</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_generate">
               <property name="text">
                <string>生成语句</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_next">
               <property name="text">
                <string>下一批图像</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_autostart">
               <property name="text">
                <string>开始自动解译</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_autostop">
               <property name="text">
                <string>停止自动解译</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="image_7">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>7.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="image_1">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>219</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>1.</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>image_1</zorder>
           <zorder>label_1</zorder>
           <zorder>label_1</zorder>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="image_2">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="images.qrc">:/img/newblack.jpg</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>2.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="actionLoadImage">
     <property name="title">
      <string>图片来源</string>
     </property>
     <addaction name="actionFromFile"/>
     <addaction name="actionFromCamera"/>
    </widget>
    <addaction name="actionLoadModel"/>
    <addaction name="actionLoadImage"/>
    <addaction name="actionSaveCaptions"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="helpmenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionInstruction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="helpmenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadModel">
   <property name="text">
    <string>网络模型</string>
   </property>
  </action>
  <action name="actionFromFile">
   <property name="text">
    <string>打开路径</string>
   </property>
  </action>
  <action name="actionFromCamera">
   <property name="text">
    <string>打开相机</string>
   </property>
  </action>
  <action name="actionSaveCaptions">
   <property name="text">
    <string>保存结果</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
